src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-dropdown.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import AlloyEditor from 'alloyeditor';
+import EzBtnImage from "./ez-btn-image";
+
+export default class EzBtnDropdown extends AlloyEditor.ButtonDropdown {
```

Comment:
In 3.0 this should be deleted

Reply:
Thanks, squashed both removals into merge commit since it's conflict resolution as well.